recipes/r-secutrialr/build.sh

```diff
@@ -0,0 +1 @@
+$R CMD INSTALL --build .
```

Comment:
this simple one could be moved inside the meta.yml as a script tag into the build section.

Reply:
like this?

```
build:
  script: $R CMD INSTALL --build .
  noarch: generic
  number: 0

```